frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -19,26 +21,65 @@ import { Loading } from '../components/loading';
 import { NavigationScreen } from '../components/navigation_screen';
 
 import { HandMarkedPaperBallot } from '../components/hand_marked_paper_ballot';
+import { TestDeckTallyReport } from '../components/test_deck_tally_report';
 
 import {
   generateTestDeckBallots,
   generateBlankBallots,
   generateOvervoteBallot,
 } from '../utils/election';
 
-interface TestDeckBallotsParams {
+interface PrecinctTallyReportParams {
```

Comment:
Minor comment: I see you were just following the existing interface naming convention! But common practice for React components is to suffix the interface name with `Props`, so `PrecincyTallyReportProps`, `HandMarkedPaperBallotsProps`